tap_appsflyer/__init__.py

```diff
@@ -427,6 +554,9 @@ def sync_in_app_events():
 def get_streams_to_sync(streams, state):
     target_stream = state.get("this_stream")
     result = streams
+    if "organic_installs" in CONFIG:
+        if CONFIG["organic_installs"]:
+            result.append(Stream("organic_installs", sync_organic_installs))
```

Comment:
@yonidavidson I've been made aware that this change cannot be used with in Stitch because our product requires stream selection to happen in the catalog v. the config.